Classes/BackEnd/AbstractEventMailForm.php

```diff
@@ -464,12 +459,12 @@ protected function addFlashMessage(FlashMessage $flashMessage)
      *
      * @param \Tx_Seminars_Model_Registration $registration
      *        the registration to which the e-mail refers
-     * @param Mail $eMail
+     * @param MailMessage $eMail
```

Comment:
This PHPDoc annotation does not any any real value anymore now that we have parameter type declarations. Please let's remove the `$eMail` annotation in all touched classes so we don't need to maintain it anymore.